core/src/test/scala/quasar/destination/redshift/RedshiftConfigSpec.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2014–2019 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.destination.redshift
+
+import quasar.blobstore.s3.{AccessKey, Bucket, Region, SecretKey}
+
+import argonaut._, Argonaut._
+
+import java.net.URI
+
+import org.specs2.mutable.Specification
+
+object RedshiftConfigSpec extends Specification {
```

Comment:
Please add tests checking that sensitive information is redacted by destination module